lib/KeyboardAwareHOC.js

```diff
@@ -273,12 +273,24 @@ function KeyboardAwareHOC(
 
     scrollToPosition = (x: number, y: number, animated: boolean = true) => {
       const responder = this.getScrollResponder()
-      responder && responder.scrollResponderScrollTo({ x, y, animated })
+      if (responder?.scrollResponderScrollTo) {
```

Comment:
I also use the Optional chaining (?.) in my repo, but it seems that we need Node 14 to support it.

So for the compatibility, we better not use it.